src/main/resources/assets/paths/datasetByIdV2.yaml

```
@@ -0,0 +1,23 @@
+get:
+  summary: Get Dataset by Id
+  description: Finds Dataset and DatasetProperties of the Dataset specified by id.
+  parameters:
+    - name: id
+      in: path
+      description: ID of the Dataset
+      required: true
+      schema:
+        type: integer
+  tags:
+    - Datasets
```

I updated the tag to the singular version for consistency in https://github.com/DataBiosphere/consent/pull/1761
```suggestion
  tags:
    - Dataset
```